packages/testing/src/api/index.ts

```diff
@@ -168,8 +171,16 @@ export function loadDatabaseRules(
         body: options.rules
       },
       (err, resp, body) => {
+        let parsedBody: LoadDatabaseRulesErrorBody
```

Comment:
Can we check if `resp.statusCode === 200` first? If the rules don't compile we return a 400. I like that we parse the response and emit the error, though, let's definitely keep that.

Reply:
okay that makes a lot of sense! Will change on Tuesday.